### ISSUE TITLE
Repurpose or return old tile servers

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#575]
Now that we have four OSMF owned servers in Europe of approximately equal capacity, we can repurpose or return the low capacity rendering servers.

I propose returning albi and scorch. They've only been used for low-zoom tiles for some time, but when they were in the regular rotation they had about 1-5% of the capacity of odin or ysera.

rhaegal would work for #565. I'm not sure what would be a good fit for necrosan with its low CPU and 8TB RAID array

## Render servers to disable at tile CDN 
- [x] albi
- [x] scorch
- [ ] rhaegal
- [ ] necrosan

**Comment by Firefishy** [comment]
@pnorman Done?

**Comment by pnorman** [comment]
No, we still have them set up. They're not in use so we can return them at any time.

**Comment by Firefishy** [comment]
Any of these suitable for https://github.com/openstreetmap/operations/issues/565?

**Comment by tomhughes** [comment]
The plan was to use rhaegal for that - it's where I ran the tests and it's why that was taken out of use as a render server before the others.

**Comment by Firefishy** [comment]
[necrosan](https://hardware.openstreetmap.org/servers/necrosan.openstreetmap.org/) seems like it should be fast and has the highest available RAM and disk. I just pushed tweaked disk queuing in https://github.com/openstreetmap/chef/commit/e74fbe00381cf087630a62f61129b6a5d9d22e8d.

**Comment by pnorman** [comment]
[rhaegal](https://hardware.openstreetmap.org/servers/rhaegal.openstreetmap.org/) is the only one still running

**Comment by pnorman** [comment]
ping @Firefishy as requested
